torchvision/transforms/v2/_misc.py

```diff
@@ -367,25 +367,17 @@ def forward(self, *inputs: Any) -> Any:
                 f"Number of boxes (shape={boxes.shape}) and number of labels (shape={labels.shape}) do not match."
             )
 
-        boxes = cast(
-            tv_tensors.BoundingBoxes,
-            F.convert_bounding_box_format(
-                boxes,
-                new_format=tv_tensors.BoundingBoxFormat.XYXY,
-            ),
+        # Alternatively we could use `boxes, valid = F.sanitize_bouding_boxes(...)` and pass both to the params dict?
```

Comment:
LMK what you think about that @pmeier

Reply:
I like the currently implemented approach better, because otherwise `_get_params` performs the actual transformation for the boxes rather than `_transform`. IIRC, we have precedence for that somewhere, but I would avoid it if we can. And it is rather easy here.

```suggestion
```